README.md

```diff
@@ -16,6 +16,23 @@ This will add the following files:
 * `app.js`
 * `package.json`
 
+### Folders
+* If a folder currently **exists**, that option will be disabled.
+
+* Otherwise you may select the folders you would like to create.
+
+#### Example
+
+```
+- Sites Folder (exists)
+
+- Components Folder (exists)
+
+- Behaviors Folder (exists)
+
+❯ ◉ Validators Folder
```

Comment:
There's no need. We're already being opinionated about what folders people should have.